src/pages/landing-page/landing-page.scss

```diff
@@ -0,0 +1,100 @@
+.landing--page {
```

Comment:
These styles are not modifiers, so I suggest to change them into Block and Element notation.

Reply:
Done. :D